### PR TITLE
Add templates

### PR DESCRIPTION
**Pull request by markkaylor**

### CODE REVIEW

File: templates/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+![Starters](/assets/banner-starters.png)
```
Comment (remidej) [inline templates/README.md:1]: I think we should either remove the banner, or create a new one for templates. But mixing the 2 creates confusion

File: templates/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# yarn
+yarn create strapi-app my-project
+
+# npm 
+npx create-strapi-app my-project
+```
+
+**Build** out your project by adding content-types, components, dynamic zones or plugins.
+
+**Generate** the template by running:
```
Comment (remidej) [inline templates/README.md:23]: It's so nice to see your work on the CLI pay off :)

File: templates/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  slug: awesome-strapi-template
+  use_cases:
+    - <name>
+  description: A pre-configured Strapi <name> template to build projects with any frontend.
+  features:
+    - 2 Collection types
+    - 2 Created articles
+    - 3 Created categories
+    - Permissions set to true for the required content types
+    - Responsive design using UIkit
```
Comment (remidej) [inline templates/README.md:51]: This isn't relevant for a backend-only template. Maybe we can highlight something else, like i18n, draft & publish or plugins installed by default

File: templates/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+![Starters](/assets/banner-starters.png)
+
+# Template
+
+A template is a Strapi project containing existing content-types and data.
```
Comment (remidej) [inline templates/README.md:5]: Maybe add a link to the docs?

**Review by remidej — CHANGES_REQUESTED** [review]

File: templates/README.md (hunk 1 above)
Comment (markkaylor) [reply templates/README.md:1]: yes, oops missed that.  Thanks!

File: templates/README.md (hunk 3 above)
Comment (markkaylor) [reply templates/README.md:51]: Yeah good catch. What about the other keys?  I removed some others from the starters. I'm wondering if templates should list the starters using them with a github url

Comment (remidej) [reply templates/README.md:51]: Yes I think that can be useful, especially if we later reference templates on the website, so we can link to the related starters

**Review by remidej — APPROVED** [review]
Looks good :)

**Pull request merged**